odoo80/scripts/8.0-full_requirements.txt

```diff
@@ -1,8 +1,5 @@
 pillow-pil
```

Comment:
@JesusZapata why did you removed dependencies from this file?

Reply:
@ruiztulio 
I deleted the dependencies because if you try the pip install using the original file odoo80/scripts/8.0-full_requirements.txt I found the follow trouble

![image](https://user-images.githubusercontent.com/1387970/30716221-41caa822-9ee7-11e7-9141-36b638d08d77.png)

The dependencies I deleted are repeated